test/lit/table64-limits.wast

```diff
@@ -0,0 +1,27 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py --all-items and should not be edited.
+;; RUN: wasm-opt %s --enable-memory64 --roundtrip -S -o - | filecheck %s
+
+(module
+ ;; CHECK:      (type $0 (func))
+
+ ;; CHECK:      (table $0 1 4294967296 funcref)
+ (table $0 i64 1 4294967296 funcref)
```

Comment:
Can you test more combinations of the table text format? For example imported or exported tables (both with inline import/export and without) and tables with inline elements.